src/helpers/request-spy.js

```diff
@@ -0,0 +1,51 @@
+import assign from 'lodash.assign';
+import sinon from 'sinon';
+
+
+const limitOffsetResponse = require('../fixtures/limit-offset-response.json');
```

Comment:
> With that said, I'm in favor of using import, load the json as a "data module", and keep it at the top alongside all the other module imports.

If it depends on toolchain, best to to use `import`. `require` immediately shoehorns consumers into browserify.